custom/sensors/NavX.java

```diff
@@ -1,7 +1,6 @@
 package org.usfirst.frc4904.standard.custom.sensors;
 
 
-import org.usfirst.frc4904.standard.LogKitten;
```

Comment:
rather than getting rid of this, maybe consider logging navx errors?

Reply:
this should probably be a diff branch